k9mail/src/main/java/com/fsck/k9/ui/messageview/MessageTopView.java

```diff
@@ -21,20 +23,19 @@
 import com.fsck.k9.mail.Message;
 import com.fsck.k9.mail.MessagingException;
 import com.fsck.k9.mailstore.MessageViewInfo;
-import com.fsck.k9.mailstore.MessageViewInfo.MessageViewContainer;
 import com.fsck.k9.view.MessageHeader;
 
 
 public class MessageTopView extends LinearLayout implements ShowPicturesController {
 
     private MessageHeader mHeaderContainer;
     private LayoutInflater mInflater;
-    private LinearLayout containerViews;
+    private FrameLayout containerView;
```

Comment:
For the future: using `ViewGroup` instead of a specific layout class allows changing the layout without having to change the code.